python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -556,44 +573,56 @@ def measureCrosstalkCoefficients(self, ratios, rejIter, rejSigma):
         """
         calib = CrosstalkCalib(nAmp=len(ratios))
 
+        if ordering is None:
+            ordering = list(ratios.keys())
+
         # Calibration stores coefficients as a numpy ndarray.
-        ordering = list(ratios.keys())
         for ii, jj in itertools.product(range(calib.nAmp), range(calib.nAmp)):
             if ii == jj:
                 values = [0.0]
             else:
                 values = np.array(ratios[ordering[ii]][ordering[jj]])
                 values = values[np.abs(values) < 1.0]  # Discard unreasonable values
 
-            calib.coeffNum[ii][jj] = len(values)
+            # Sigma clip.
+            if ii != jj:
+                for rej in range(rejIter):
+                    if len(values) == 0:
+                        break
+                    lo, med, hi = np.percentile(values, [25.0, 50.0, 75.0])
+                    sigma = 0.741*(hi - lo)
```

Comment:
Make a note of this is sigma from the IQR, with 0.741 assuming a normal distribution.